doc/services/ipc/ipc_service/ipc_service.rst

```diff
@@ -0,0 +1,192 @@
+.. _ipc_service:
+
+IPC service
+###########
+
+.. contents::
+    :local:
+    :depth: 2
+
+The IPC service API provides an interface to exchange data between two domains
+or CPUs. It must be ensured that the receiving domain (or CPU) is aware that
+the data has arrived.
```

Comment:
Is this a requirement? @emob-nordic

Reply:
It might sound too hard. We currently use mbox for pinging, but this is specific to a backend.. I will remove this sentence. It's not applicable for every backend we can imagine.